share/man/kodev.1

```diff
@@ -0,0 +1,130 @@
+.TH KODEV 1
+.SH NAME
+kodev \- Kore project management tool
+
+.SH SYNOPSIS
+.BR kodev
+[\fIOPTION\fR] ...
+
+.SH DESCRIPTION
+This documentation describes the application management tools for building and
+running a kore project. For information regarding the technical interface and
+C bindings, look to
+.BR KORE(3)
+where these functions and structures are described.
+.BR kore
+projects may be managed using the following OPTIONS;
+
+.BR create
+.RS
+Create a new application skeleton with the name that is passed to it. This will
+create a new directory with all the files required to begin hacking about.
+This will also generate a self-signed certificate \fIunless\fR \fBkore\fR was
+built with \fBNOTLS=1\fR. Note this certificate should not be used for
+production.
+.RE
+
+.BR build
+.RS
+Build the application. The properties of this build are read from
+.BR conf/build.conf
+and the semantics of which are discussed in \fBCONFIGURATION FILES\fR
+.RE
+
+.BR run
+.RS
+Run an application (\-fnr implied). The current directory is the path of the
```

Comment:
I think we should clarify that kodev run will either run the kore binary (from PREFIX) or the application binary (if single-binary was turned on).